recipes/wpa-supplicant/wpa-supplicant.inc

```diff
@@ -9,17 +9,18 @@ inherit make c++ pkgconfig
 
 DEPENDS_LIBC = "libdl"
 DEPENDS_LIBC:HOST_LIBC_mingw = ""
-DEPENDS += "libdbus-1 libnl-3 libnl-genl-3 libnl-genl-3 libgnutls libgcrypt \
-	libgpg-error ${DEPENDS_LIBC}"
+DEPENDS += "libnl-3 libnl-genl-3 libnl-genl-3 librt \
+	    ${CRYPTO} ${DEPENDS_LIBC}"
 
 SRC_URI = "http://hostap.epitest.fi/releases/wpa_supplicant-${PV}.tar.gz \
-           file://defconfig-gnutls \
            file://wpa-supplicant.sh \
            file://wpa_supplicant.conf \
            file://wpa_supplicant.conf-sane \
 	   "
 
-S = "${SRCDIR}/wpa_supplicant-${PV}/wpa_supplicant"
+S = "${SRCDIR}/wpa_supplicant-${PV}"
+
+WPADIR = "wpa_supplicant"
 
```

Comment:
core/conf/oelayout.conf:S                = "${SRCDIR}/${P}"
do you really need to override ${S} ?